<?xml version="1.0"?>
<launch>
	<node pkg="darknet_ros" name="darknet_ros_wrapper" type="darknet_ros_node">
		<param name="name_file" value="$(find darknet_ros)/darknet/data/voc.names"/>
		<param name="weights_file" value="$(find darknet_ros)/tiny-yolo-voc.weights"/>
		<param name="cfg_file" value="$(find darknet_ros)/darknet/cfg/tiny-yolo-voc.cfg"/>
		<param name="thresh" value="0.5"/>
		<remap from="/camera/image_raw" to="/camera/rgb/image_raw"/>
	</node>
</launch>
